exporter/exporterhelper/queued_retry.go

```diff
@@ -167,6 +195,8 @@ func (qrs *queuedRetrySender) send(req request) error {
 
 // shutdown is invoked during service shutdown.
 func (qrs *queuedRetrySender) shutdown() {
+	metricproducer.GlobalManager().DeleteProducer(r)
```

Comment:
Don't delete this, because there may be other instances using it. just "upsertentry" with a function that returns 0.